### ISSUE TITLE
[Bug]: Unable to download podcast episode - downloadPodcastEpisode: Error Error: ffmpeg exited with code 234

### ISSUE THREAD

**Issue by neilsimon** [advplyr/audiobookshelf#2858]
### Describe the issue

I added Never Post podcast to my library a while ago, and all other episodes have worked correctly, but the latest does not. The error as in the subject was followed by the following log line:
```
Full stderr dump for episode url "https://chrt.fm/track/3G82GG/media.transistor.fm/4797700d/de9cdc1a.mp3": ffmpeg version 6.1.1 Copyright (c) 2000-2023 the FFmpeg developers built with gcc 13.2.1 (Alpine 13.2.1_git20231014) 20231014 configuration: --prefix=/usr --disable-librtmp --disable-lzma --disable-static --disable-stripping --enable-avfilter --enable-gpl --enable-ladspa --enable-libaom --enable-libass --enable-libbluray --enable-libdav1d --enable-libdrm --enable-libfontconfig --enable-libfreetype --enable-libfribidi --enable-libharfbuzz --enable-libmp3lame --enable-libopenmpt --enable-libopus --enable-libplacebo --enable-libpulse --enable-librav1e --enable-librist --enable-libsoxr --enable-libsrt --enable-libssh --enable-libtheora --enable-libv4l2 --enable-libvidstab --enable-libvorbis --enable-libvpx --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxcb --enable-libxml2 --enable-libxvid --enable-libzimg --enable-libzmq --enable-lto=auto --enable-lv2 --enable-openssl --enable-pic --enable-postproc --enable-pthreads --enable-shared --enable-vaapi --enable-vdpau --enable-version3 --enable-vulkan --optflags=-O3 --enable-libjxl --enable-libsvtav1 --enable-libvpl libavutil 58. 29.100 / 58. 29.100 libavcodec 60. 31.102 / 60. 31.102 libavformat 60. 16.100 / 60. 16.100 libavdevice 60. 3.100 / 60. 3.100 libavfilter 9. 12.100 / 9. 12.100 libswscale 7. 5.100 / 7. 5.100 libswresample 5. 0.100 / 5. 0.100 libpostproc 57. 3.100 / 57. 3.100 Splitting the commandline. Reading option '-i' ... matched as output url with argument 'pipe:0'. Reading option '-y' ... matched as option 'y' (overwrite output files) with argument '1'. Reading option '-loglevel' ... matched as option 'loglevel' (set logging level) with argument 'debug'. Reading option '-c' ... matched as option 'c' (codec name) with argument 'copy'. Reading option '-metadata' ... matched as option 'metadata' (add metadata) with argument 'podcast=1'. Reading option '-metadata' ... matched as option 'metadata' (add metadata) with argument 'album=Never Post'. Reading option '-metadata' ... matched as option 'metadata' (add metadata) with argument 'album-sort=Never Post'. Reading option '-metadata' ... matched as option 'metadata' (add metadata) with argument 'artist=Charts & Leisure'. Reading option '-metadata' ... matched as option 'metadata' (add metadata) with argument 'artist-sort=Charts & Leisure'. Reading option '-metadata' ... matched as option 'metadata' (add metadata) with argument 'comment=<p>Contributing Producer Marie Kilaru discusses the emotional impacts of making and seeing Before and After posts; Charlie Harding of Switched on Pop joins Jason to talk about what pop-musical acts have sounded like the internet. Also: Georgia reads Franny Choi’s “Unrequited Lovesong for the Panopticon”</p><p>-<br />Become a member at <a href="https://www.neverpo.st/#/portal/">https://www.neverpo.st/</a></p><p>Join us <a href="https://twitch.tv/mikerugnetta">on Twitch</a>, Friday, April 12th @ 2pm ET to chat about audio production!</p><p>–</p><ul><li>Call us at <strong>651 615 5007</strong> to leave a voice mail</li><li>Drop us a voice memo via <a href="https://airtable.com/appIvXY8gz5ikRbF0/pagjGcjdYg2alw8cD/form">airtable</a></li><li>Or email us at theneverpost at gmail dot com</li></ul><p>–</p><p><strong>Intro Links</strong></p><ul><li><a href="https://www.reuters.com/technology/cybersecurity/why-near-miss-cyberattack-put-us-officials-tech-industry-edge-2024-04-05/">Why a near-miss cyberattack put US officials and the tech industry on edge</a> – Reuters</li><li><a href="https://kansasreflector.com/2024/04/04/when-facebook-fails-local-media-matters-even-more-for-our-planets-future/">When Facebook fails, local media matters even more for our planet’s future</a> – Kansas Reflector</li><li><a href="https://www.thehandbasket.co/p/meta-the-handbasket-censorship-slay-giants">We can slay giants</a> – Handbasket</li><li><a href="https://www.reuters.com/technology/fcc-vote-restore-net-neutrality-rules-reversing-trump-2024-04-02/">FCC to vote to restore net neutrality rules, reversing Trump</a> – Reuters</li><li><a href="https://techcrunch.com/2024/03/16/linkedin-wants-to-add-gaming-to-its-platform/?guccounter=1">LinkedIn plans to add gaming to its platform</a> – Techcrunch</li><li><a href="https://www.twitch.tv/mikerugnetta">Mike’s Twitch</a></li><li><a href="https://podcasts.apple.com/us/podcast/the-dust-bowl-who-is-to-blame/id1468678544?i=1000650458540">Mike on the Alarmist!</a></li></ul><p>–</p><p><strong>Before and After Photos</strong></p><ul><li>Find Marie at <a href="https://mariekilaru.com/">her website</a> and <a href="https://twitter.com/mariekilaru">on X</a>.</li><li>Mark Gaetano is on <a href="https://www.instagram.com/m.a.rk/?hl=en">IG</a> and <a href="https://www.youtube.com/@snarkymarky/videos">YT</a></li></ul><p>Special thanks to Hannah Meacock-Ross for editorial support</p><p>–</p><p><strong>What (Pop Music) Does the Internet Sound Like?</strong></p><ul><li>You should listen to <a href="https://switchedonpop.com/">Switched on Pop</a> generally, but also here’s <a href="https://switchedonpop.com/episodes/100-gecs-interview">the interview they did with 100 gecs</a>.</li></ul><p>–<br /><a href="https://www.nytimes.com/interactive/2020/01/03/opinion/franny-choi-privacy-poem.html">Unrequited Love Song for the Panopticon</a> was used with permission </p><p>–</p><p>Never Post’s producers are Audrey Evans, Georgia Hampton and The Mysterious Dr. Firstname Lastname. Our contributing producer this episode is Marie Kilaru. Our senior producer is Hans Buetow. Our executive producer is Jason Oberholtzer. The show’s host is Mike Rugnetta. </p><p><em>After the accident we had</em></p><p><em>the phrase after the accident.</em></p><p><br /></p><p><em>Also this: before the accident.</em></p><p><em>We had a drawer marked</em></p><p><br /></p><p><em>before and after, and after</em></p><p><em>and before happenings</em></p><p><br /></p><p><em>we'd add atrocities and</em></p><p><em>incidents and the wild</em></p><p><br /></p><p><em>asters someone before</em></p><p><em>and after keeps leaving.</em></p><p><br /></p><p><a href="https://www.poetryfoundation.org/poems/154967/after-5fcfd71e226a3">After</a> By Andrea Cohen</p><p><br /></p><p>Never Post is a production of Charts &amp; Leisure.</p> <br /><p>Check out Join the Party wherever you get podcasts and also at <a href="https://www.jointhepartypod.com/">jointhepartypod.com</a>.</p> <strong> <a href="https://www.neverpo.st/#/portal/signup">★ Support this podcast ★</a> </strong>'. Reading option '-metadata' ... matched as option 'metadata' (add metadata) with argument 'genre=Society & Culture;Technology'. Reading option '-metadata' ... matched as option 'metadata' (add metadata) with argument 'language=en'. Reading option '-metadata' ... matched as option 'metadata' (add metadata) with argument 'MVNM=Never Post'. Reading option '-metadata' ... matched as option 'metadata' (add metadata) with argument 'MVIN=6'. Reading option '-metadata' ... matched as option 'metadata' (add metadata) with argument 'track=6'. Reading option '-metadata' ... matched as option 'metadata' (add metadata) with argument 'series-part=6'. Reading option '-metadata' ... matched as option 'metadata' (add metadata) with argument 'title=Switched on Post'. Reading option '-metadata' ... matched as option 'metadata' (add metadata) with argument 'title-sort=Switched on Post'. Reading option '-metadata' ... matched as option 'metadata' (add metadata) with argument 'year=2024'. Reading option '-metadata' ... matched as option 'metadata' (add metadata) with argument 'date=Wed, 10 Apr 2024 12:30:00 -0400'. Reading option '-metadata' ... matched as option 'metadata' (add metadata) with argument 'releasedate=Wed, 10 Apr 2024 12:30:00 -0400'. Reading option '-metadata' ... matched as option 'metadata' (add metadata) with argument 'podcast-type=episodic'. Reading option '/audiobooks/podcasts/Never Post/Switched on Post (f2ad49f3-06e3-47c6-a7a4-681ac39ec771).mp3' ... matched as output url. Finished splitting the commandline. Parsing a group of options: global . Applying option y (overwrite output files) with argument 1. Applying option loglevel (set logging level) with argument debug. Successfully parsed a group of options. Parsing a group of options: input url pipe:0. Successfully parsed a group of options. Opening an input file: pipe:0. [AVFormatContext @ 0x7ff0b421b7c0] Opening 'pipe:0' for reading [pipe @ 0x7ff0aff3ed40] Setting default whitelist 'crypto,data' [mp3 @ 0x7ff0b421b7c0] Format mp3 probed with size=131072 and score=51 id3v2 ver:3 flags:00 len:97979 [mp3 @ 0x7ff0b421b7c0] invalid concatenated file detected - using bitrate for duration [mp3 @ 0x7ff0b421b7c0] pad 576 1585 [mp3 @ 0x7ff0b421b7c0] Skipping 0 bytes of junk at 98406. [mp3 @ 0x7ff0b421b7c0] Before avformat_find_stream_info() pos: 98406 bytes read:131072 seeks:0 nb_streams:2 [mjpeg @ 0x7ff0b457f8c0] marker=eb avail_size_in_buf=96601 [mjpeg @ 0x7ff0b457f8c0] marker parser used 7822 bytes (62576 bits) [mjpeg @ 0x7ff0b457f8c0] marker=e8 avail_size_in_buf=88690 [mjpeg @ 0x7ff0b457f8c0] marker parser used 37715 bytes (301720 bits) [mjpeg @ 0x7ff0b457f8c0] marker=d9 avail_size_in_buf=49832 [mjpeg @ 0x7ff0b457f8c0] Found EOI before any SOF, ignoring [mjpeg @ 0x7ff0b457f8c0] marker parser used 0 bytes (0 bits) [mjpeg @ 0x7ff0b457f8c0] marker=cf avail_size_in_buf=49414 [mjpeg @ 0x7ff0b457f8c0] marker parser used 0 bytes (0 bits) [mjpeg @ 0x7ff0b457f8c0] marker=c5 avail_size_in_buf=47910 [mjpeg @ 0x7ff0b457f8c0] marker parser used 0 bytes (0 bits) [mjpeg @ 0x7ff0b457f8c0] marker=c2 avail_size_in_buf=45728 [mjpeg @ 0x7ff0b457f8c0] bits 203 is invalid [mp3 @ 0x7ff0b421b7c0] demuxer injecting skip 1105 / discard 0 [mp3float @ 0x7ff0b47478c0] skip 1105 / discard 0 samples due to side data [mp3float @ 0x7ff0b47478c0] skip 1105/1152 samples [mp3 @ 0x7ff0b421b7c0] max_analyze_duration 5000000 reached at 5015510 microseconds st:0 [mp3 @ 0x7ff0b421b7c0] Could not find codec parameters for stream 1 (Video: mjpeg (Progressive), 1 reference frame, none(bt470bg/unknown/unknown, center)): unspecified size Consider increasing the value for the 'analyzeduration' (0) and 'probesize' (5000000) options [mp3 @ 0x7ff0b421b7c0] After avformat_find_stream_info() pos: 180326 bytes read:184533 seeks:0 frames:195 Input #0, mp3, from 'pipe:0': Metadata: encoder : Lavf60.3.100 title : Switched on Post artist : Charts & Leisure album : Never Post track : 7 TGID : 92a8df76-c990-4672-89ee-df798940264c genre : Podcast date : 2024 Duration: N/A, start: 0.025057, bitrate: 128 kb/s Chapters: Chapter #0:0: start 0.000000, end 575.000000 Metadata: title : Intro Chapter #0:1: start 575.000000, end 755.965000 Metadata: title : Unrequited Love Song for the Panopticon Chapter #0:2: start 755.965000, end 1803.976000 Metadata: title : Before-and-After Images Chapter #0:3: start 1803.976000, end 1862.976000 Metadata: title : Unrequited Love Song for the Panopticon, pt. 2 Chapter #0:4: start 1862.976000, end 3348.976000 Metadata: title : Who Sounds Like the Internet? Chapter #0:5: start 3348.976000, end 3413.976000 Metadata: title : Unrequited Love Song for the Panopticon, pt. 3 Chapter #0:6: start 3413.976000, end 3529.143000 Metadata: title : Credits Stream #0:0, 194, 1/14112000: Audio: mp3, 44100 Hz, stereo, fltp, 128 kb/s Metadata: encoder : Lavc60.3. Stream #0:1, 1, 1/90000: Video: mjpeg (Progressive), 1 reference frame, none(bt470bg/unknown/unknown, center), 90k tbr, 90k tbn (attached pic) Metadata: title : Cover comment : Cover (front) Successfully opened the file. Parsing a group of options: output url /audiobooks/podcasts/Never Post/Switched on Post (f2ad49f3-06e3-47c6-a7a4-681ac39ec771).mp3. Applying option c (codec name) with argument copy. Applying option metadata (add metadata) with argument podcast=1. Applying option metadata (add metadata) with argument album=Never Post. Applying option metadata (add metadata) with argument album-sort=Never Post. Applying option metadata (add metadata) with argument artist=Charts & Leisure. Applying option metadata (add metadata) with argument artist-sort=Charts & Leisure. Applying option metadata (add metadata) with argument comment=<p>Contributing Producer Marie Kilaru discusses the emotional impacts of making and seeing Before and After posts; Charlie Harding of Switched on Pop joins Jason to talk about what pop-musical acts have sounded like the internet. Also: Georgia reads Franny Choi’s “Unrequited Lovesong for the Panopticon”</p><p>-<br />Become a member at <a href="https://www.neverpo.st/#/portal/">https://www.neverpo.st/</a></p><p>Join us <a href="https://twitch.tv/mikerugnetta">on Twitch</a>, Friday, April 12th @ 2pm ET to chat about audio production!</p><p>–</p><ul><li>Call us at <strong>651 615 5007</strong> to leave a voice mail</li><li>Drop us a voice memo via <a href="https://airtable.com/appIvXY8gz5ikRbF0/pagjGcjdYg2alw8cD/form">airtable</a></li><li>Or email us at theneverpost at gmail dot com</li></ul><p>–</p><p><strong>Intro Links</strong></p><ul><li><a href="https://www.reuters.com/technology/cybersecurity/why-near-miss-cyberattack-put-us-officials-tech-industry-edge-2024-04-05/">Why a near-miss cyberattack put US officials and the tech industry on edge</a> – Reuters</li><li><a href="https://kansasreflector.com/2024/04/04/when-facebook-fails-local-media-matters-even-more-for-our-planets-future/">When Facebook fails, local media matters even more for our planet’s future</a> – Kansas Reflector</li><li><a href="https://www.thehandbasket.co/p/meta-the-handbasket-censorship-slay-giants">We can slay giants</a> – Handbasket</li><li><a href="https://www.reuters.com/technology/fcc-vote-restore-net-neutrality-rules-reversing-trump-2024-04-02/">FCC to vote to restore net neutrality rules, reversing Trump</a> – Reuters</li><li><a href="https://techcrunch.com/2024/03/16/linkedin-wants-to-add-gaming-to-its-platform/?guccounter=1">LinkedIn plans to add gaming to its platform</a> – Techcrunch</li><li><a href="https://www.twitch.tv/mikerugnetta">Mike’s Twitch</a></li><li><a href="https://podcasts.apple.com/us/podcast/the-dust-bowl-who-is-to-blame/id1468678544?i=1000650458540">Mike on the Alarmist!</a></li></ul><p>–</p><p><strong>Before and After Photos</strong></p><ul><li>Find Marie at <a href="https://mariekilaru.com/">her website</a> and <a href="https://twitter.com/mariekilaru">on X</a>.</li><li>Mark Gaetano is on <a href="https://www.instagram.com/m.a.rk/?hl=en">IG</a> and <a href="https://www.youtube.com/@snarkymarky/videos">YT</a></li></ul><p>Special thanks to Hannah Meacock-Ross for editorial support</p><p>–</p><p><strong>What (Pop Music) Does the Internet Sound Like?</strong></p><ul><li>You should listen to <a href="https://switchedonpop.com/">Switched on Pop</a> generally, but also here’s <a href="https://switchedonpop.com/episodes/100-gecs-interview">the interview they did with 100 gecs</a>.</li></ul><p>–<br /><a href="https://www.nytimes.com/interactive/2020/01/03/opinion/franny-choi-privacy-poem.html">Unrequited Love Song for the Panopticon</a> was used with permission </p><p>–</p><p>Never Post’s producers are Audrey Evans, Georgia Hampton and The Mysterious Dr. Firstname Lastname. Our contributing producer this episode is Marie Kilaru. Our senior producer is Hans Buetow. Our executive producer is Jason Oberholtzer. The show’s host is Mike Rugnetta. </p><p><em>After the accident we had</em></p><p><em>the phrase after the accident.</em></p><p><br /></p><p><em>Also this: before the accident.</em></p><p><em>We had a drawer marked</em></p><p><br /></p><p><em>before and after, and after</em></p><p><em>and before happenings</em></p><p><br /></p><p><em>we'd add atrocities and</em></p><p><em>incidents and the wild</em></p><p><br /></p><p><em>asters someone before</em></p><p><em>and after keeps leaving.</em></p><p><br /></p><p><a href="https://www.poetryfoundation.org/poems/154967/after-5fcfd71e226a3">After</a> By Andrea Cohen</p><p><br /></p><p>Never Post is a production of Charts &amp; Leisure.</p> <br /><p>Check out Join the Party wherever you get podcasts and also at <a href="https://www.jointhepartypod.com/">jointhepartypod.com</a>.</p> <strong> <a href="https://www.neverpo.st/#/portal/signup">★ Support this podcast ★</a> </strong>. Applying option metadata (add metadata) with argument genre=Society & Culture;Technology. Applying option metadata (add metadata) with argument language=en. Applying option metadata (add metadata) with argument MVNM=Never Post. Applying option metadata (add metadata) with argument MVIN=6. Applying option metadata (add metadata) with argument track=6. Applying option metadata (add metadata) with argument series-part=6. Applying option metadata (add metadata) with argument title=Switched on Post. Applying option metadata (add metadata) with argument title-sort=Switched on Post. Applying option metadata (add metadata) with argument year=2024. Applying option metadata (add metadata) with argument date=Wed, 10 Apr 2024 12:30:00 -0400. Applying option metadata (add metadata) with argument releasedate=Wed, 10 Apr 2024 12:30:00 -0400. Applying option metadata (add metadata) with argument podcast-type=episodic. Successfully parsed a group of options. Opening an output file: /audiobooks/podcasts/Never Post/Switched on Post (f2ad49f3-06e3-47c6-a7a4-681ac39ec771).mp3. [out#0/mp3 @ 0x7ff0b0333700] No explicit maps, mapping streams automatically... [vost#0:0/copy @ 0x7ff0b0186880] Created video stream from input stream 0:1 [aost#0:1/copy @ 0x7ff0aefd5100] Created audio stream from input stream 0:0 [file @ 0x7ff0b5a59ec0] Setting default whitelist 'file,crypto,data' [mp3 @ 0x7ff0b0186600] dimensions not set [out#0/mp3 @ 0x7ff0b0333700] Could not write header (incorrect codec parameters ?): Invalid argument Error opening output file /audiobooks/podcasts/Never Post/Switched on Post (f2ad49f3-06e3-47c6-a7a4-681ac39ec771).mp3. Error opening output files: Invalid argument [AVIOContext @ 0x7ff0ae9be040] Statistics: 0 bytes written, 0 seeks, 0 writeouts [AVIOContext @ 0x7ff0aed9c980] Statistics: 184533 bytes read, 0 seeks 
```

### Steps to reproduce the issue

1. Add Never Post podcast to library (feed: https://feeds.transistor.fm/never-post)
2. Download the episodes. It will likely fail on `Switched on Post`.

### Audiobookshelf version

v2.8.1

### How are you running audiobookshelf?

Docker

**Comment by advplyr** [comment]
I saw that this had to do with the cover image embedded in the audio files. The most recent episode "Mailbag, Episodes 4-6" at this time does not have an embedded cover so I was able to download it.

I fixed this issue by only including the audio streams when these get downloaded. When manually downloading those episodes with embedded covers they weren't displaying on Windows.

As a side note in testing this issue I came across a bug downloading episode "OO🚫⛔❌ H M🙅💥🕳️ GO🛑🔇💃" on Windows. The node fs is not able to find this file after it gets downloaded. This seems to be a known bug in node and is fixed in v20.12.0 https://github.com/nodejs/node/issues/48673 so it will be fixed in the next release of Abs as well.
